pytest_inmanta/plugin.py

```diff
@@ -1137,6 +1137,14 @@ def enumerate_model(model: str):
         for key, blob in exporter._file_store.items():
             self.add_blob(key, blob)
 
+        for key, value in inmanta.resources.resource._resources.items():
+            name_value = value[1].get("name")
+            if name_value == "id":
```

Comment:
Let's make the variable names a bit more concrete:
```suggestion
        for cls_name, value in inmanta.resources.resource._resources.items():
            name_id_attribute = value[1]["name"]
            if name_id_attribute == "id":
```